lib/src/package_config_impl.dart

```diff
@@ -77,7 +77,7 @@ class SimplePackageConfig implements PackageConfig {
       var name = package.name;
       if (packageNames.contains(name)) {
         onError(PackageConfigArgumentError(
-            name, "packages", "Duplicate package name"));
+            name, "packages", "Duplicate package $name"));
```

Comment:
Should be part of the `toString` of the `PackageConfigArgumentError` error generally?

Reply:
Not sure what you are asking. Answer is probably "no" 😉 .

The `PackageConfigArgumentError` is just an `ArgumentError` which implements `PackageConfigError` (just so you can catch all `PackageConfigError`s).
It doesn't have specific fields for "package name errors".

Here it's just an argument error, the argument in error is `packages`, the problem description is that `packages` contains two entries with the same package name (now `$name`).